applications/nrf_desktop/doc/ble_scan.rst

```diff
@@ -135,7 +135,7 @@ a. The scanning is stopped and the |NCS|'s :ref:`nrf_bt_scan_readme` automatical
 
 .. important::
    If a Bluetooth peer is already connected with a 1-ms connection interval, the next peer is connected with a 10-ms connection interval instead of 7.5 ms.
-   The peer is connected with a 10-ms connection interval also in case :kconfig:`CONFIG_BT_MAX_CONN` is set to value greater than 2 and :kconfig:`CONFIG_CAF_BLE_USE_LLPM` Kconfig option is enabled.
+   The peer is connected with a 10 ms connection interval also in case :kconfig:`CONFIG_BT_MAX_CONN` is set to value greater than 2 and :kconfig:`CONFIG_CAF_BLE_USE_LLPM` Kconfig option is enabled.
```

Comment:
```suggestion
   The peer is connected with a 10-ms connection interval also in case :kconfig:`CONFIG_BT_MAX_CONN` is set to value greater than 2 and :kconfig:`CONFIG_CAF_BLE_USE_LLPM` Kconfig option is enabled.
```